src/sentry_tracing.c

```diff
@@ -0,0 +1,31 @@
+#include "sentry_sync.h"
+
+sentry_value_t
+sentry__span_get_trace_context(sentry_value_t span)
+{
+    if (sentry_value_is_null(span)
+        || sentry_value_is_null(sentry_value_get_by_key(span, "trace_id"))
+        || sentry_value_is_null(sentry_value_get_by_key(span, "span_id"))) {
+        return sentry_value_new_null();
+    }
+
+    sentry_value_t trace_context = sentry_value_new_object();
+
+#define PLACE_VALUE(Key, Source)                                               \
```

Comment:
Can you also add a corresponding `#undef PLACE_VALUE` after the method please.
It looks like this is also missing in `sentry_scope.c`, must have been an oversight.